### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by sam-nguyen7**
Update Readme to reflect current state of operator

### CODE REVIEW

**Comment by clcollins** [comment]
Doc change,

/lgtm

File: README.md [hunk 1]
```diff
@@ -30,9 +30,40 @@ To test a new version of the operator locally using CRC you need to:
...
+spec:
+  subjectKind: Group
+  subjectName: dedicated-admins
+  clusterPermissions:
+    - dedicated-admins-cluster
+  permissions:
+    - 
+      clusterRoleName: dedicated-admins-project
+      namespacesAllowedRegex: ".*"
+      namespacesDeniedRegex: "(^kube-.|^openshift.*|^ops-health-monitoring$|^management-infra$|^default$|^logging$|^sre-app-check$)"
```
Comment (jharrington22) [inline README.md]: The regex here needs updating specifically `^kube-.*`

File: README.md [hunk 2]
```diff
@@ -30,9 +30,40 @@ To test a new version of the operator locally using CRC you need to:
...
+      allowFirst: true
+    - 
+      clusterRoleName: admin 
+      namespacesAllowedRegex: ".*" 
+      namespacesDeniedRegex: "(^kube-.|^openshift.*|^ops-health-monitoring$|^management-infra$|^default$|^logging$|^sre-app-check$)"
```
Comment (jharrington22) [inline README.md]: here too

**Pull request merged**